src/olympia/devhub/templates/devhub/includes/source_form_field.html

```diff
@@ -1,6 +1,6 @@
 <div class="binary-source{% if not field.errors %} edit_initially_hidden{% endif %}">
   <p>
-    {{ _('If your add-on contains binary or obfuscated code other than known '
+    {{ _('If your add-on contains minified, concatenated or otherwise machine-generated code other than known '
```

Comment:
I think we should keep "binary" as I believe we are only replacing instances of "obfuscated", so this should be "If your add-on contains binary, minified, concatenated or otherwise machine-generated code...".

Reply:
@jvillalobos for copy confirmation

Reply:
I think it's okay without "binary", since that's something only legacy extensions could use. WebExtensions can talk to binaries, but not ones contained inside the XPI.

Reply:
Okay, thanks for the clarification.

@ManishAradwad please change it back to what you had before.